web/themes/custom/recipe_theme/src/sass/content/_node.landing-page.scss

```diff
@@ -3,3 +3,31 @@
   margin-bottom: 3rem;
   opacity: 0.5;
 }
+
+.landing-page__body {
+  text-align: center;
+  @include for-phone-only {
+    margin-left: 2rem;
+    margin-right: 2rem;
+  }
+  @include for-tablet-portrait-up {
+    margin-left: 3rem;
+    margin-right: 3rem;
+  }
+  @include for-tablet-landscape-up {
+    margin-left: 4rem;
+    margin-right: 4rem;
+  }
+  @include for-desktop-up {
+    margin-left: 5rem;
+    margin-right: 5rem;
+  }
+  @include for-big-desktop-up {
+    margin-left: 6rem;
+    margin-right: 6rem;
+  }
+}
+
+.landing-page img {
```

Comment:
Please use nesting everywhere for the sake of consistency.

```sass
.outer-selector {
  .inner-selector {
    ...rules
  }
}
```

Reply:
Fixed.